packages/block-library/src/navigation/edit.js

```diff
@@ -85,12 +85,14 @@ function Navigation( {
 		{
 			allowedBlocks: ALLOWED_BLOCKS,
 			orientation: attributes.orientation || 'horizontal',
-			renderAppender:
+			appender:
 				( isImmediateParentOfSelectedBlock &&
 					! selectedBlockHasDescendants ) ||
-				isSelected
-					? InnerBlocks.DefaultAppender
-					: false,
+				isSelected ? (
+					<InnerBlocks.DefaultAppender />
```

Comment:
Same here with the name of the component.

Reply:
Also fixed here: https://github.com/WordPress/gutenberg/commit/67c2525c25dceef43900aeaf8fedf8b2a4d758a0